src/vs/base/common/network.ts

```diff
@@ -241,12 +269,12 @@ class FileAccessImpl {
 		return uri;
 	}
 
-	private toUri(uriOrModule: URI | string, moduleIdToUrl?: { toUrl(moduleId: string): string }): URI {
+	private toUri(uriOrModule: URI | string, moduleIdToUrl: { toUrl(moduleId: string): string }): URI {
```

Comment:
It looks like we no longer pass anything in from the outside that implements `toUrl` other than the global `require` that we now depend on, so why not remove `moduleIdToUrl` parameter altogether if it is always `require`?

Reply:
The gymnastics here are needed for the monaco editor. `asFileUri` and `asBrowserUri` are unused in the entirety of the monaco editor and will therefore be removed via tree-shaking. By removing them, all references to `require` are deleted from `network.ts` and then webpack can ingest the file. Otherwise, if we have a remaining `require` usage, webpack complains. Here's an example build https://github.com/microsoft/vscode/actions/runs/3421361753/jobs/5697403051#step:15:15